ansible/Vagrantfile

```diff
@@ -11,6 +11,22 @@ Vagrant.configure(2) do |config|
     vb.customize ["modifyvm", :id, "--memory", "2048"]
   end
 
+  config.vm.provider :openstack do |os|
+    config.ssh.username = "centos"
```

Comment:
Does adding `node.ssh.username = "vagrant"` to just the `config.vm.provider "virtualbox" do |vb|` also work?